frontend/src/components/EditAttributes.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <template v-for="(attr, rowIndex) in attributes" :key="rowIndex">
+    <TextInput label="Name" v-model="attributes[rowIndex].name"
+               :args="{ id: `initialattrName${rowIndex}`, list: 'attributes' }"
+               @change="onChange()">
+      <template v-slot:datalist>
+        <option v-for="(attr, index) in availableFieldNames" :key="index" :value="attr.name">
+          {{ `${attr.name} (${attr.type})` }}
+        </option>
+      </template>
+    </TextInput>
+    <Select label="Type" v-model="attributes[rowIndex].type" @change="onChange()"
+            :args="{ id: `initialattrType${rowIndex}`, disabled: !attr.isNew }"
+            :options="
+              Object.keys(ATTR_TYPES_NAMES).map((type) => {
+                return { value: type, text: ATTR_TYPES_NAMES[type] };
+              })"/>
```

Comment:
I guess there's problem somewhere in here: when looking at schema details it displays types of all fields as `string` even if API returns other type